lib/simulator-xcode-6.js

```diff
@@ -192,20 +189,12 @@ class SimulatorXcode6 {
     log.info(`starting simulator with command: open ${args.join(' ')}`);
     await exec('open', args, {timeout: STARTUP_TIMEOUT});
 
-    //wait for it to be 'booted'
-    let checkBooted = async function () {
+    // wait for the simulator to be 'booted'
+    await retryInterval(parseInt(BOOTING_TIMOUT/BOOTED_POLLING_INTERVAL), BOOTED_POLLING_INTERVAL, async () => {
```

Comment:
Interesting. Didn't know that. https://github.com/appium/appium-ios-simulator/pull/24